src/payloads/services/index.ts

```diff
@@ -74,11 +78,41 @@ export const getServicesTuple = (
   return { payload, isJson: true };
 };
 
+const specialServicesPreferenceFactory: Map<string, () => boolean> = new Map<
+  string,
+  () => boolean
+>([["cgn", isCgnActivated]]);
+
 export const getServicesPreferences = (
   services: ReadonlyArray<ServicePublic>
 ) =>
   new Map<ServiceId, ServicePreference>(
     services.map(s => {
+      const metadata = s.service_metadata;
+      if (metadata && SpecialServiceMetadata.is(metadata)) {
+        const hasSpecialServiceInbox = fromNullable(
+          metadata.custom_special_flow
+        )
+          .chain(csf =>
+            fromNullable(specialServicesPreferenceFactory.get(csf)).map(h =>
+              h()
+            )
+          )
+          .getOrElse(false);
+        return [
+          s.service_id,
+          {
+            is_inbox_enabled: hasSpecialServiceInbox,
+            is_email_enabled: hasSpecialServiceInbox
+              ? faker.datatype.boolean()
```

Comment:
```suggestion
              ? getRandomValue(false, faker.datatype.boolean(), "services")
```
we are trying to dominate the randomness in the dev-server with the configuration
A dev could go crazy if a value change on every request :dizzy_face: